### DISCUSSION TITLE
I need a vertical line separator

### DISCUSSION THREAD

**Discussion by Bogdan-Torkhov** [nvim-lualine/lualine.nvim#819] [Q&A]
Hello,i need a vertical line separator like on this [photo](https://user-images.githubusercontent.com/41551030/103467925-32372b00-4d54-11eb-88d6-6d39c46854d8.png) the | symbol don't work like on photo.What i need to do?

**Comment by fitrh** [comment]
> the | symbol don't work like on the photo

Can you describe more what do you mean by **don't work**? The screenshot is the default configuration, so unless you are changed the components separator config, I think you should get the same look.

Maybe it's your font(?)

**Comment by shadmansaleh** [comment]
You need to set the component separator option.

```lua
require'lualine'.setup {
  options = { component_separators = '|' }
}
```
